astroquery/mpc/tests/test_mpc.py

```diff
@@ -0,0 +1,31 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import os
+import re
+
+import pytest
+
+from ... import mpc
+from ...utils.testing_tools import MockResponse
+from ...utils import commons
+from ...exceptions import TableParseError
+
+class MockResponseMPC(MockResponse):
+
+    def __init__(self, **kwargs):
+        super(MockResponseMPC, self).__init__(**kwargs)
+
+def read_data_file(filename):
+    query_args = dict()
+    with open(os.path.join(os.getcwd(), 'tests/data', filename), 'r') as data:
```

Comment:
You'll need to use something like:
```
DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
```
instead of `os.getcwd()`, since it is possible to run these tests from any arbitrary location.  `os.path.dirname(__file__)` will return the path to this file, which should be fixed relative to the test data.

Reply:
Oh, I actually ended up not using the data file--I'll remove the related items. Good catch, though!